ui/cypress/fixtures/kubernetes/volumes.json

```diff
@@ -1144,6 +1144,118 @@
+            },
+            "creationTimestamp": "2021-04-21T07:51:02Z",
+            "finalizers": [
+                "storage.metalk8s.scality.com/volume-protection"
+            ],
+            "generation": 2,
+            "managedFields": [
+                {
+                    "apiVersion": "storage.metalk8s.scality.com/v1alpha1",
+                    "fieldsType": "FieldsV1",
+                    "fieldsV1": {
+                        "f:metadata": {
+                            "f:annotations": {
+                                ".": {},
+                                "f:kubectl.kubernetes.io/last-applied-configuration": {}
+                            }
+                        },
+                        "f:spec": {
+                            ".": {},
+                            "f:lvmLogicalVolume": {
+                                ".": {},
+                                "f:size": {},
+                                "f:vgName": {}
+                            },
+                            "f:nodeName": {},
+                            "f:storageClassName": {},
+                            "f:template": {
+                                ".": {},
+                                "f:metadata": {
+                                    ".": {},
+                                    "f:labels": {
+                                        ".": {},
+                                        "f:app.kubernetes.io/name": {},
+                                        "f:app.kubernetes.io/part-of": {}
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "manager": "kubectl-client-side-apply",
+                    "operation": "Update",
+                    "time": "2021-04-21T07:51:02Z"
+                },
+                {
+                    "apiVersion": "storage.metalk8s.scality.com/v1alpha1",
+                    "fieldsType": "FieldsV1",
+                    "fieldsV1": {
+                        "f:metadata": {
+                            "f:finalizers": {}
+                        },
+                        "f:spec": {
+                            "f:mode": {},
+                            "f:template": {
+                                "f:metadata": {
+                                    "f:creationTimestamp": {}
+                                },
+                                "f:spec": {}
+                            }
+                        },
+                        "f:status": {
+                            ".": {},
+                            "f:conditions": {},
+                            "f:deviceName": {}
+                        }
+                    },
+                    "manager": "storage-operator",
+                    "operation": "Update",
+                    "time": "2021-04-21T07:52:07Z"
+                }
+            ],
+            "name": "mongodb-volume",
```

Comment:
Would make sense to add this object in the same commit that updates the test for using it.